demos/toolbar/toolbar.scss

```diff
@@ -54,14 +54,14 @@
 
 .demo-toolbar--custom {
   @include mdc-toolbar-fill-color-accessible($material-color-red-900);
-}
 
-.demo-toolbar__menu-icon--custom,
-.demo-toolbar__icon--custom {
-  @include mdc-toolbar-icon-ink-color($material-color-red-100);
+  .demo-toolbar__menu-icon--custom,
```

Comment:
This was broken due to css specificity. Our toolbar generates `.mdc-toolbar .mdc-toolbar__icon`, but this was producing just `.demo-toolbar__icon--custom`.